src/js/jsx/sections/libraries/assets/Graphic.jsx

```diff
@@ -54,17 +57,51 @@ define(function (require, exports, module) {
             });
         },
 
+        /**
+         * Handle add layer from graphic asset
+         * @private
+         */
         _handleAdd: function () {
-            this.getFlux().actions.libraries.createLayerFromElement(this.props.element);
+            // TODO: new graphic is add to a fixed location. Ideally it should place the new graphic layer 
+            // at the center of the view port.
+            this.getFlux().actions.libraries.createLayerFromElement(this.props.element, { x: 100, y: 100 });
+        },
+        
+        /**
+         * Create preview of the dragged graphic asset under the cursor.
+         * @private
+         * @return {ReactComponent?}
+         */
+        _renderDragPreview: function () {
+            if (!this.props.dragPosition) {
```

Comment:
Right now, when I let go of the mouse, the drag preview stays on screen. Can you fix that?